samples/bluetooth/llpm/README.rst

```diff
@@ -80,7 +80,7 @@ Requirements
   * |nRF52840DK|
   * |nRF52DK|
   * |nRF5340DK|
-  * Other boards running BLE Controller variants that support LLPM (see :ref:`nrfxlib:ble_controller` Proprietary feature support)
+  * Other boards running BLE Controller variants that support LLPM (see :ref:`nrfxlib:softdevice_controller` Proprietary feature support)
```

Comment:
```suggestion
  * Other boards running Bluetooth LE Controller variants that support LLPM (see :ref:`nrfxlib:softdevice_controller` Proprietary feature support)
```

Reply:
There is also a mention of "BLE controller" in line 26 - that could be replaced with "SoftDevice Controller" imo.

Reply:
BLE in line 20 also.

Reply:
Fixed

Reply:
Still BLE in line 20, or do you wish to fix this outside of this PR?

Reply:
We are using BLE in multiple places in this repo. I would suggest to take that in a separate PR